.github/dependabot.yml

```diff
@@ -4,11 +4,11 @@ version: 2
   updates:
```

Comment:
Why are "updates" intended with 2 white spaces "  " ? Is that significant.

The example github generates does not have those indentation.

Reply:
Right, we missed that in #2016. Check https://jsonformatter.org/yaml-validator